helm_deploy/prisoner-offender-search/values.yaml

```diff
@@ -36,6 +36,10 @@ generic-service:
           deny all;
           return 401;
         }
+        location /prisoner-differences/delete {
```

Comment:
endpoint needs to be called from cronjob so ingress protected so can only be called from inside namespace